polkadot/node/core/approval-voting/src/import.rs

```diff
@@ -856,6 +872,15 @@ pub(crate) mod tests {
 					si_tx.send(Ok(Some(ExecutorParams::default()))).unwrap();
 				}
 			);
+
+			assert_matches!(
+				handle.recv().await,
+				AllMessages::RuntimeApi(
+					RuntimeApiMessage::Request(_, RuntimeApiRequest::NodeFeatures(_, si_tx), )
+				) => {
+					si_tx.send(Ok(NodeFeatures::EMPTY)).unwrap();
```

Comment:
should we have some tests that return the enabled feature?

Reply:
Good point, done!